firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLitePersistence.java

```diff
@@ -54,9 +54,8 @@
  * helper routines that make dealing with SQLite much more pleasant.
  */
 public final class SQLitePersistence extends Persistence {
-  enum DataMigration {
-    BuildOverlays
-  }
+  /** Constant string to indicate a data migration is required to support overlays. */
+  public static String DATA_MIGRATION_BUILD_OVERLAYS = "BUILD_OVERLAYS";
```

Comment:
It might make sense to move this to `Persistence` since there is a corresponding `MemoryOverlayMigrationManager`.

Reply:
Done.